scripts/templates/helper.py

```diff
@@ -988,7 +988,7 @@ def get_pfntables(specs, meta, namespace, tags):
 def get_pfncbtables(specs, meta, namespace, tags):
     tables = []
     for cname in sorted(meta['class'], key=lambda x: meta['class'][x]['ordinal']):
-        objs = get_class_function_objs(specs, cname, 1.0)
+        objs = get_class_function_objs(specs, cname)
```

Comment:
Adding APIs released after 1.0 will change the size and layout of ze_callbacks_t which will be a backwards compatibility break.  That is why I originally limited this to 1.0